src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -2077,13 +2118,19 @@ private protected override void CheckLabel(RoleMappedData examples, out int weig
     }
 
     /// <summary>
-    /// Train logistic regression using a parallel stochastic gradient method.
+    /// The <see cref="IEstimator{TTransformer}"/> for training logistic regression using a parallel stochastic gradient method.
     /// </summary>
+    /// <remarks>
+    /// The Stochastic Gradient Descent (SGD) is one of the popular stochastic optimization procedures that can be integrated
+    /// into several machine learning tasks to achieve state-of-the-art performance. This trainer implements SGD for binary classification
+    /// that supports multi-threading without any locking. If the associated optimization problem is sparse, it achieves a nearly optimal
+    /// rate of convergence. For more details, please refer to http://arxiv.org/pdf/1106.5730v2.pdf.
```

Comment:
>e. For more details [](start = 26, length = 19)

Call out that this uses the Hogwild method.

Reply:
I though we were reducing Hogwild from our API names. This used to be called HogwildSgdTrainer. So I didn't mention Hogwild here and the only mentioning is in the reference. Do you want Hogwild to be mentioned in the text here?

---
In reply to: [259456602](https://github.com/dotnet/machinelearning/pull/2688#discussion_r259456602) [](ancestors = 259456602)

Reply:
I think we don't want it in the API name so that we're free to change implementations, but I think it's good to be explicit in the docs on what style of SGD this is performing so that advanced users can take it into account. Also, just pointing to Arxiv is a bit confusing because it looks like "for information about SGD, look here" and then it's a paper on a very specific multithreaded implementation.

---
In reply to: [259480809](https://github.com/dotnet/machinelearning/pull/2688#discussion_r259480809) [](ancestors = 259480809,259456602)

Reply:
mentioned Hogwild.

---
In reply to: [259495627](https://github.com/dotnet/machinelearning/pull/2688#discussion_r259495627) [](ancestors = 259495627,259480809,259456602)